Workbooks/Azure Active Directory Conditional Access/Conditional Access Insights/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+    "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
+    "name":"Conditional Access Insights (Preview)",
+    "author": "Microsoft",
+    "isPreview": true,
+    "galleries": [{ "type": "workbook", "resourceType": "microsoft.aadiam/tenant", "order": 100 }]
```

Comment:
Looks like another template at this level also has order 100. Make the order unique.